client/app/services/usage-graphs/usage-graphs.spec.js

```diff
@@ -56,4 +56,42 @@ describe('Usage Graph component', () => {
         expect(graphs[1].innerHTML).to.eq("6");
         expect(graphs[2].innerHTML).to.eq("38");
   })
+
```

Comment:
not sure why eslint didn't pick up this space 🤔 

we might not check .spec.js files (probs not gonna guess)